src/lib/components/compositions/checkbox/animated.story.svelte

```diff
@@ -26,22 +26,40 @@
       let root = el.$$.root;
       console.log(root);
       let errorEl = root.querySelector('input');
-      errorEl.setAttribute('aria-invalid', true);
+      errorEl?.setAttribute('aria-invalid', 'true');
     }
   });
 
-  $: { console.log(errorMessage) }
+  $: {
+    console.log(errorMessage);
+  }
 </script>
 
 {#if withForm}
   <Form onSubmitWhenValid={submitForm}>
-    <Checkbox bind:this={el} {checkValidity} {errorMessage} {key} {onChange} {value}>
+    <Checkbox
+      bind:this={el}
+      {checkValidity}
+      {errorMessage}
+      firstInvalidInput={false}
```

Comment:
No functional change here, just adding so that `story` files are type correct. You'll see a bunch of these.